WordPress/gutenberg
# Better inline paste

Fixes #2459, and inline Editable paste.

* The pasteHandler now really handles all paste. Before, the Editable component handled block paste and inline paste.
* We pass `{ inline: true }` if the editable expects inline content. If so, we strip everything that is not text or inline tags.
* Images are now seen as non inline content, so a block will be created.

To test:

* Copy an image from the old editor (click, copy), then paste it in a paragraph. An image block should be created.
* Copy some content that is not all inline. This can be some blocks, or some content from a site. Paste it in a caption or another single line Editable. The content should be pasted in the caption, but only as text and formatting.
